tests/integrations/litestar/test_litestar_websockets.py

```diff
@@ -27,14 +26,21 @@
 
 
 @asynccontextmanager
-async def dishka_app(view, provider) -> AsyncGenerator[TestClient, None]:
+async def dishka_app_via_setup(view, provider) -> AsyncGenerator[TestClient, None]:
     app = Litestar([view], debug=True)
     container = make_async_container(provider)
     setup_dishka(container, app)
-    async with LifespanManager(app):
-        yield TestClient(app)
+    with TestClient(app) as client:
+        yield client
```

Comment:
I can revert those changes if you prefer, but since litestar's testclient will run the lifespan manager for you there is no need to use that library